front/migrations/db/migration_111.sql

```diff
@@ -0,0 +1,7 @@
+-- Migration created on Nov 14, 2024
+ALTER TABLE
+    "public"."content_fragments"
+ADD
+    COLUMN "sId" VARCHAR(255);
+
+CREATE INDEX "content_fragments_s_id" ON "content_fragments" ("sId");
```

Comment:
I believe it's more efficient to apply the index after the backfill. (and you might want to apply it `concurrently`).

Reply:
The backfill does rely on "where sId is Null" so I felt it might be useful to have the index from the beginning no ?

Will add `concurrently` indeed 👍

Reply:
> The backfill does rely on "where sId is Null" so I felt it might be useful to have the index from the beginning no ?

There are 186,898 content fragment.
If you don't have an index before hand, read will cost you 186 full tables scan (batch of 1000), write will cost you almost nothing.
If you have the index before hand, you'll save on the reads but write will cost 186k indexes update.

I don't think it matter a lot, just sharing my context.

Reply:
Fair point ! agree backfill latency doesn't matter much, will move the index creation to migration 112 👍